auth/tenant_mgt.go

```diff
@@ -240,6 +242,11 @@ func (t *TenantToCreate) EnableEmailLinkSignIn(enable bool) *TenantToCreate {
 	return t.set(enableEmailLinkSignInKey, enable)
 }
 
+// EnableAnonymousUsers enables or disables anonymous user.
```

Comment:
Our auth docs use the term "anonymous authentication."  Should that be used here?

I'd suggest that phrase, or, alternatively, "anonymous user access."